sdk/storage/azure-storage-blob/azure/storage/blob/_blob_utils.py

```diff
@@ -0,0 +1,668 @@
+# -------------------------------------------------------------------------
```

Comment:
I feel like the file name is a little bit misleading. _blob_utils is like a private thing, while, customers are expected to call down load method directly from this file, do we want to rename the file or move the content to a different place?

Reply:
This file is internal, and wont be used by customers - I have an issue opened to refactor the contents in additional modules as needed.

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/6030